src/pure/utils.ts

```diff
@@ -106,19 +115,26 @@ export async function spawnVitestVersion(command: string, args: string[], env?:
     env,
   })
 
-  child.on('error', () => {
-    log.info('Command not found')
-  })
+  return new Promise((resolve) => {
+    child.stdout.on('data', (data) => {
+      child.kill()
+      log.info(data.toString())
 
-  for await (const line of chunksToLinesAsync(child.stdout)) {
-    child.kill()
-    log.info(line)
+      const match = data.toString().match(/vitest\/(\d+.\d+.\d+)/)
 
-    const match = line.match(/vitest\/(\d+.\d+.\d+)/)
+      if (match && match.length > 0)
+        resolve(match[1])
+    });
 
-    if (match && match.length > 0)
-      return match[1]
-  }
+    child.on('error', () => {
+      log.info('Command not found')
+      resolve(undefined);
+    })
+
+    child.on('close', (code) => {
+      resolve(undefined);
+    });
+  });
```

Comment:
For a reason I did not investigate further, `chunksToLinesAsync(child.stdout)` returns empty here, and the version detection fails. I replaced the lib call with a plain promise. This implementation should work on every system. I am also resolving everything to `undefined` in the other cases because of the behavior described on the tests.